New playtest page (#27)
OpenRA/OpenRAWebsiteV3: news/_posts/2022-10-30-playtest-20221030.md

---
kind: article
title: "Playtest 20221030"
author: "PunkPun"
created_at: 2022-10-30 22:00 +0000
disqus_id: "playtest-20221030"

> permalink: "/news/playtest-20221030/"

Reviewer: Dates need to be updated.

---

> Welcome back, Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It is finally here!

Reviewer: Same here and for the image names.

There have been a ton of refactors under the hood -- it took us a very long time to catch and squash all those pesky bugs -- and we are excited to finally showcase a number of new features and improvements that have been built up over the last year and a half.

One highlight feature is the new hierarchical pathfinder, which massively increases performance on large or more complex maps.

<figure>
  <img src="{{ '/images/news/20221031-hpf.png' | relative_url }}" style="width: 600px" alt="HPF helps units better predict obstacles ahead of them" />
  <figcaption>HPF helps units to better predict obstacles ahead of them.</figcaption>
</figure>

> Our networking code has had some major improvements in both low and high latency games.

Reviewer: Maybe
```suggestion
Our new netcode brings major improvements in both low and high latency games. Clients that experience connection hiccups can now catch up and do not slow the game permanently.
```
if you want more than one sentence. (It still irks me a bit that this reads like the netcode has improvements during games... while it is the connection that sees those improvements.)

Author: it's not just the connection though. The way we handle the delay is different

Author: it's not just about the hiccups, the whole game just plainly runs smoother and is more responsive

Reviewer: Then mention this too. How is an end-user supposed to know this from the single sentence?

Author: Sure

<figure>
  <video width="700" autoplay loop muted>
    <source src="{{ '/images/news/20221031-td.mp4' | relative_url }}" type="video/mp4">
    <source src="{{ '/images/news/20221031-td.ogv' | relative_url }}" type="video/ogg">
    <img src="{{ '/images/news/20221031-td.png' | relative_url }}" width="700" />
  </video>
</figure>

We have added transient text notifications matching speech notifications. Now you can play without sound!

<figure>
  <video width="700" autoplay loop muted>

>     <source src="{{ '/images/news/20221031-transient.mp4' | relative_url }}" type="video/mp4">

Reviewer: ```suggestion
    <source src="{{ '/images/news/20221031-transient.mov' | relative_url }}" type="video/mp4">
```
?

  </video>
  <figcaption>OpenRA now supports tilting on slopes!</figcaption>
</figure>

<br>

Some of the more visible new features and improvements include:
 * Measures against spamming the lobby and in-game chat
 * The ability to mute players
 * Support for disabling the chat in replays
 * New Red Alert missions
 * An ingame encyclopedia in Dune 2000

Notable changes for mappers and modders include:
 * Automatic map reloading in the background
 * A change to the way speed is calculated for ground units
 * A new map format which includes the map preview (minimap) in the map versioning (map hash)
 * Loaders for more audio formats (e.g., mp3 and ogg), and formats used by the remasters
 * Many linting and documentation updates

As well as some other changes worth mentioning:
 * Toolchain and changes to support the .NET 6 runtime
 * A massive amount of upgrades to both ingame and client UI
 * Many more hotkeys became rebindable
 * Substantial progress towards localisation

All in all, 615 pull requests were merged since last release and 552 issues were closed bringing the total number of open tickets down by 141.

> This post covered just a small subset of all the changes and improvements added since the last release. We encourage anybody interested in finding out more to read the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog). Head on over to the [download page]({{ '/download' | relative_url }}) to install the new playtest and let us know what you think in the comments below, on [our forum](https://forum.openra.net/), [community Discord server](https://discord.openra.net) or on [GitHub](https://github.com/OpenRA/OpenRA/issues). Happy playtesting!

Reviewer: https://github.com/OpenRA/OpenRA/wiki/Changelog/1e2499c01b75fc76018dfe96ce915379557b3716 is the full changelog link.

---
kind: article
title: "Playtest 20221030"
author: "PunkPun, Therapist"
created_at: 2022-10-30 22:00 +0000
disqus_id: "playtest-20221030"
permalink: "/news/playtest-20221030/"
---

> Welcome back Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It's finally here!!

Author: This really kills the energy

> Welcome back Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It's finally here!!

Reviewer: Not sure what you mean, we still keep one `!`. Not having the comma and keeping two exclamation marks is bad style.

> Welcome back Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It's finally here!!

Reviewer: (If you insist we can keep the second `!`, but I'd still add the comma.)

> Welcome back Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It's finally here!!

Author: I'm mostly talking about the comma. Adding it literally kills the tempo

> Welcome back Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It's finally here!!

Reviewer: Apparently this isn't just bad style but plain incorrect:
> You need a comma after “welcome” if it is an introductory interjection or if it is immediately followed by the name or another word directly addressing the people being spoken to.

Whether or not it kills tempo is subjective, and the EVA announcement also has it if you look at the end of the video: https://www.youtube.com/watch?v=JkWkYZgLk2I

> Welcome back Commanders! After more than a year and a half the OpenRA team proudly presents [Playtest 20221030]({{ '/download' | relative_url }}). It's finally here!!

Author: sure...

> There's been a ton of refactors under the hood -- it took us a very long time to catch and squash all those pesky bugs -- and we're excited to finally showcase a number of new features that've been built up over the years!

Author: Honestly I dunno how best to rewrite it. I've implemented the second part of the suggestionI don't know how to fix the first part

> One of the highlight features is the new hierarchical pathfinder. It should increase performance massively on large or more complex maps.

Author: I'd rather have complex (not claustrophobic. Implemented other suggestion

<figure>
  <img src="{{ '/images/news/20221031-hpf.png' | relative_url }}" style="width: 600px" alt="HPF helps units better predict obstacles ahead of them" />

>   <figcaption>HPF helps units better predict obstacles ahead of them.</figcaption>

Author: The sentence doesn't feel whole with this change

>   <figcaption>HPF helps units better predict obstacles ahead of them.</figcaption>

Reviewer: The sentence isn't really whole (i.e. correct English) before this change. If you want to keep `better` then it should read
```suggestion
  <figcaption>HPF helps units to better predict obstacles ahead of them.</figcaption>
```

</figure>

> Our netcode has had some major improvements in both low and high latency games.

Author: This kills the energy

> Our netcode has had some major improvements in both low and high latency games.

Reviewer: I'm not sure what you mean by that. If anything this increases the energy by using active voice? (And fits the rest of the article better.)

> Our netcode has had some major improvements in both low and high latency games.

Author: The way you phrase it makes it sound like I need a second sentence or image to follow it up with. But really I want this to be a standalone statement

> Our netcode has had some major improvements in both low and high latency games.

Reviewer: Yes, it probably should. Having a single disconnected sentence feels a bit weird to me.

<figure>
  <video width="700" autoplay loop muted>
    <source src="{{ '/images/news/20221031-td.mp4' | relative_url }}" type="video/mp4">
    <source src="{{ '/images/news/20221031-td.ogv' | relative_url }}" type="video/ogg">
    <img src="{{ '/images/news/20221031-td.png' | relative_url }}" width="700" />
  </video>

>   <figcaption>Forces collide</figcaption>

Reviewer: ```suggestion
  <figcaption>Forces collide.</figcaption>
```

</figure>

> We've added transient text notifications matching speech notifications. Now you can play without sound!

Reviewer: ```suggestion
We added transient text notifications to match speech notifications. Now you can play without sound!
```

We could include something here along the lines of "Now it is harder than ever to miss important notifications during battle" to give this a spin that it doesn't just benefit people who play without sound. (nts: revisit)

Author: Perhaps

<figure>
  <video width="700" autoplay loop muted>
    <source src="{{ '/images/news/20221031-transient.mp4' | relative_url }}" type="video/mp4">
    <source src="{{ '/images/news/20221031-transient.ogv' | relative_url }}" type="video/ogg">
    <img src="{{ '/images/news/20221031-transient.png' | relative_url }}" width="700" />
  </video>
  <figcaption>Nuke incomming!</figcaption>
</figure>

The fairly recent Dune film has brought in many players into Dune 2k and a decently sized multiplayer community was born. With the feedback from the community Dune 2k has received a number of balance and polish changes.

<figure>
  <video width="700" autoplay loop muted>
    <source src="{{ '/images/news/20221031-dune.mp4' | relative_url }}" type="video/mp4">
    <source src="{{ '/images/news/20221031-dune.ogv' | relative_url }}" type="video/ogg">
    <img src="{{ '/images/news/20221031-dune.png' | relative_url }}" width="700" />
  </video>

>   <figcaption>Storm is approaching!</figcaption>

Reviewer: That reads weird though, Imo.

>   <figcaption>Storm is approaching!</figcaption>

Author: removed it

</figure>

Tiberian Sun has also received its fair share of features and bug fixes. This time around the highlight is tilting on slopes.

<figure>
  <video width="700" autoplay loop muted>
    <source src="{{ '/images/news/20221031-slope.mp4' | relative_url }}" type="video/mp4">
    <source src="{{ '/images/news/20221031-slope.ogv' | relative_url }}" type="video/ogg">
    <img src="{{ '/images/news/20221031-slope.png' | relative_url }}" width="700" />
  </video>

>   <figcaption>OpenRA now supports tilting on slopes!</figcaption>

Reviewer: ```suggestion
  <figcaption>OpenRA now supports tilting on slopes.</figcaption>
```

</figure>

<br>

Some of the more visible new features and improvements include:
 * Measures against spamming in-game chat
 * The ability to mute players
 * Support for disabling the chat in replays
 * New Red Alert missions
 * An ingame encyclopedia in Dune 2000

Notable changes for mappers and modders include:
 * Automatic map reloading in the background

>  * Changed the way speed is calculated for ground units

Reviewer: ```suggestion
 * A change to the way speed is calculated for ground units
```
(nts: revisit)

 * A new map format which includes the map preview (minimap) in the map versioning (map hash)
 * Loaders for more audio formats (e.g. mp3, ogg), and formats used by the remasters
 * Many linting and documentation updates

As well as some other changes worth mentioning:

>  * Toolchain and runtime changes to .NET 6

Reviewer: ```suggestion
 * Toolchain changes to support the .NET 6 runtime
```
(nts: revisit?)

>  * Toolchain and changes to support .NET 6 runtime

Reviewer: You missed the article
```suggestion
 * Toolchain and changes to support the .NET 6 runtime
```

 * A massive amount of upgrades to both ingame and client UI

>  * Many more hotkeys are now rebindable

Author: We didn't really add hotkeys, just made them rebindable

>  * Many more hotkeys are now rebindable

Author: changed the time

---
kind: article
title: "Playtest 20221030"
author: "PunkPun"
created_at: 2022-10-30 22:00 +0000
disqus_id: "playtest-20221030"
permalink: "/news/playtest-20221030/"
---

After an over a year long wait, [Playtest 20221030]({{ '/download' | relative_url }}) is finally released!

This time around there's been a ton of refactors under the hood, and it took us a very a long time to catch and squash all those pesky bugs. But that's not all that happened, over the years many features have built up and we are excited to finally showcase them!

Added transient text notifications matching speech notifications. Now you can play without sound!

<figure>
  <video width="700" autoplay loop muted>
    <source src="{{ '/images/news/20221031-transient.mp4' | relative_url }}" type="video/mp4">
    <source src="{{ '/images/news/20221031-transient.ogv' | relative_url }}" type="video/ogg">
    <img src="{{ '/images/news/20221031-transient.png' | relative_url }}" width="700" />
  </video>
  <figcaption>Nuke incomming!</figcaption>
</figure>

One of the highlight features is the new hierarchical pathfinder. It should increase performance massively on large or more complex maps.

<figure>

>   <img src="{{ '/images/news/20221031-hpf.png' | relative_url }}" style="width: 600px" alt="HPF helps units better predict obsticles ahead of them" />
>   <figcaption>HPF helps units better predict obsticles ahead of them.</figcaption>

Reviewer 2: ```suggestion
  <img src="{{ '/images/news/20221031-hpf.png' | relative_url }}" style="width: 600px" alt="HPF helps units better predict obstacles ahead of them" />
  <figcaption>HPF helps units better predict obstacles ahead of them.</figcaption>
```

  </video>
  <figcaption>Storm is approaching!</figcaption>
</figure>

INTERMISSION: some cool stats
 * 565 pull requests were merged since last release
 * 533 issues were closed, bringing the total amount down by 136

TS has also received its fair share of features and bug fixes. This time around the highlight is tilting on slopes.

<figure>
  <video width="700" autoplay loop muted>
    <source src="{{ '/images/news/20221031-slope.mp4' | relative_url }}" type="video/mp4">
    <source src="{{ '/images/news/20221031-slope.ogv' | relative_url }}" type="video/ogg">
    <img src="{{ '/images/news/20221031-slope.png' | relative_url }}" width="700" />
  </video>
  <figcaption>OpenRA now supports tilting on slopes!</figcaption>
</figure>

<br>

Some of the more visible new features and improvements include:
 * New Red Alert missions
 * An ingame encyclopedia in Dune 2000
 * Measures against spamming in-game chat
 * The ability to mute players
 * Support for disabling the chat in replays

Notable changes for mappers and modders include:
 * Automatic map reloading in the background
 * Changed the way speed is calculated for ground units

>  * New map format which includes the map preview in the map versioning (map hash)

Author: It's for mappers and modders, it's fine if it's a bit more technical

 * Loaders for more audio formats (e.g. mp3, ogg), and formats used by the remasters
 * Many linting and documentation updates

As well as some other changes worth mentioning:
 * Toolchain and runtime changes to .NET 6
 * A massive amount of upgrades to both ingame and client UI
 * Many more hotkeys are now rebindable
 * Substantial progress towards localisation

> This post has covered just a small subset of the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)), and we encourage anybody interested in finding out more to read the full list. Head on over to the [download page]({{ '/download' | relative_url }}) to install the new playtest, and let us know what you think in the comments below, on [our forum](https://forum.openra.net/), [community Discord server](https://discord.gg/UcUztfr), or [GitHub](https://github.com/OpenRA/OpenRA/issues).

Reviewer 2: Next stable is https://github.com/OpenRA/OpenRA/wiki/Changelog not bleed.

Reviewer 2: community Discord server is https://discord.openra.net

Author: > Next stable is https://github.com/OpenRA/OpenRA/wiki/Changelog not bleed.

Yes, I'll update it once we have that changelog

> community Discord server is https://discord.openra.net/

Good catch, I just copied it over from another page
